### DISCUSSION TITLE
properties overridable at runtime

### DISCUSSION THREAD

**Discussion by abdelkader-k** [quarkusio/quarkus#22923] [Q&A]
### Describe the bug

How to define properties overridable at runtime

**Comment by iocanel** [comment]
Properties configured with `@ConfigProperty` are overridable at runtime by default. 

For example:

```
   @ConfigProperty(name="greeting")
   String greeting;

```

And then set the `GREETING` environment variable before you run your app:

```
GREETING=Ciao java -jar ./target/quarkus-app/quarkus-run.jar
```
